SUPPORT.md

```diff
@@ -20,6 +20,8 @@ Every community member is welcome to participate, help to triage and answer the
 - [WireMock Community Slack](http://slack.wiremock.org/) provides the `#help` channel for Q&A
 - [As a question on StackOverflow](https://stackoverflow.com/questions/tagged/wiremock), use the `wiremock` tag
 
+If you're looking for material to help you practice using WireMock on your own machine, in your own time, [Bas Dijkstra](https://www.linkedin.com/in/basdijkstra/) maintains an open source workshop on WireMock that is free for all to use: https://github.com/basdijkstra/wiremock-workshop
```

Comment:
I would prefer to keep the _WireMock Community_ localized to the community resources, but indeed it would make sense to put it somewhere on the website. I moved it to the text down, and I suggest adding it to https://wiremock.org/external-resources/ as a _Trainings/Workshops_ section

```suggestion
```